dataplane/common/src/lib.rs

```diff
@@ -40,3 +40,39 @@ pub struct BackendList {
 
 #[cfg(feature = "user")]
 unsafe impl aya::Pod for BackendList {}
+
+#[derive(Copy, Clone, Debug)]
+#[repr(C)]
+pub struct ClientKey {
+    pub ip: u32,
```

Comment:
I know we don't have an issue for it yet but at some point we'll need to think about v6 here :)